src/libraries/Common/src/Interop/Unix/System.Native/Interop.ReadStdinUnbuffered.cs

```diff
@@ -8,8 +8,8 @@ internal static partial class Interop
 {
     internal static partial class Sys
     {
-        [DllImport(Libraries.SystemNative, EntryPoint = "SystemNative_ReadStdin", SetLastError = true)]
-        internal static extern unsafe int ReadStdin(byte* buffer, int bufferSize);
+        [GeneratedDllImport(Libraries.SystemNative, EntryPoint = "SystemNative_ReadStdin", SetLastError = true)]
+        internal static unsafe partial int ReadStdin(byte* buffer, int bufferSize);
 
         [DllImport(Libraries.SystemNative, EntryPoint = "SystemNative_InitializeConsoleBeforeRead")]
         internal static extern void InitializeConsoleBeforeRead(byte minChars = 1, byte decisecondsTimeout = 0);
```

Comment:
Why aren't we using `GeneratedDllImport` on `InitializeConsoleBeforeRead` and `UninitializeConsoleAfterRead`?

Reply:
We only targeted the ones that had non-blittable types or had settings that would require runtime handling (`SetLastError=true` for example), since the others would already be inline-able.